elements/text.js

```diff
@@ -0,0 +1,11 @@
+let BaseElement = require('./base.element');
```

Comment:
Rename file to textBox.js to make it more clear.

Reply:
Done